urlbox/urlbox-python
# fix(repo): replace .io with .com, update failing test when running vi…

#### What's this PR do?

Replaces occurrences of .io with .com

#### Background context

This is due to Urlbox's migration from .io to .com. 

#### How should this be manually tested?

Ensure python installed on host computer, run pytest in root dir and ensure passing.
